pkg/alertmanager/alertmanager_metrics_test.go

```diff
@@ -950,6 +1021,8 @@ type nflogMetrics struct {
 	queryErrorsTotal        prometheus.Counter
 	queryDuration           prometheus.Histogram
 	propagatedMessagesTotal prometheus.Counter
+	maintenanceTotal        prometheus.Counter
```

Comment:
Changes in the rest of the file are copied from their respective `prometheus` files. There are comments pointing to the source above each relevant section.